### PR TITLE
completed css-grid-classes | Janelle Gatmaitan

### PR DESCRIPTION
**Pull request by JanelleGatmaitan**
image

### CODE REVIEW

File: css-grid-classes/styles.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  width: 33%;
+}
+
+.column-two-third {
+  width: 33%;
+}
+
+.column-three-third {
+  width: 33%;
+}
```
Comment (codingcodymiller) [inline css-grid-classes/styles.css]: These should be removed. Just reuse the `column-one-third` class.

File: css-grid-classes/styles.css [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  width: 25%;
+}
+
+.column-three-fourth {
+  width: 25%;
+}
+
+.column-four-fourth {
+  width: 25%;
+}
```
Comment (codingcodymiller) [inline css-grid-classes/styles.css]: These should be removed.

File: css-grid-classes/styles.css [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  width: 30%;
+}
+
+.column-one-halfB {
+  width: 30%;
+}
+
+.column-two-halfB {
+  width: 70%;
+}
```
Comment (codingcodymiller) [inline css-grid-classes/styles.css]: These should be removed.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Janelle, great code so far! Please see my notes

**Comment by JanelleGatmaitan** [comment]
image

**Review by codingcodymiller — APPROVED** [review]
Nicely done!! 👍

**Pull request merged**